tests/xfstests/install.pm

```diff
@@ -33,6 +34,12 @@ sub log_create {
     assert_script_run($cmd);
 }
 
+sub collect_version {
+    my $file = shift;
+    my $cmd  = "cd /tmp/xfstests-dev; git log | head -n 1 | awk '{print $2}' > $file; cd -; rpm -qa xfsprogs xfsdump btrfsprogs >> $file; uname -r >> $file";
```

Comment:
_git rev-parse HEAD_ is a better way to get the commit ID
```suggestion
    my $cmd  = "cd /tmp/xfstests-dev; git rev-parse HEAD > $file; cd -; rpm -qa xfsprogs xfsdump btrfsprogs >> $file; uname -r >> $file";
```

Reply:
Thanks, modified.